vendor/bat-native-ledger/include/bat/ledger/ledger_client.h

```diff
@@ -161,10 +170,35 @@ class LEDGER_EXPORT LedgerClient {
       int line,
       int vlog_level) const = 0;
 
+  // Logs confirmations-related debug information
+  virtual std::unique_ptr<confirmations::LogStream> LogConfirmations(
+      const char* file,
+      int line,
+      const int log_level) const = 0;
+
   virtual void OnRestorePublishers(ledger::OnRestoreCallback callback) = 0;
 
   virtual void SaveNormalizedPublisherList(
     const ledger::PublisherInfoListStruct& normalized_list) = 0;
+
+  virtual void URLRequest(
+      const std::string& url,
+      const std::vector<std::string>& headers,
+      const std::string& content,
+      const std::string& content_type,
+      const ledger::URL_METHOD method,
+      ledger::URLRequestCallback callback) = 0;
+
+  virtual void SaveConfirmationsState(const std::string& name,
+                                      const std::string& value,
+                                      ledger::OnSaveCallback callback) = 0;
+  virtual void LoadConfirmationsState(const std::string& name,
+                                      ledger::OnLoadCallback callback) = 0;
+  virtual void ResetConfirmationsState(const std::string& name,
+                                       ledger::OnResetCallback callback) = 0;
+  virtual uint32_t SetConfirmationsTimer(const uint64_t time_offset) = 0;
+  virtual void KillConfirmationsTimer(uint32_t timer_id) = 0;
+  virtual void SetConfirmationsIsReady(const bool is_ready) = 0;
```

Comment:
why is this both a Ledger and LedgerClient method? I'm very confused about what it's supposed to do